signac/contrib/project.py

```diff
@@ -710,10 +709,8 @@ def _find_job_ids(self, filter=None):
         if not filter:
             return list(self._job_dirs())
         filter = dict(parse_filter(_add_prefix("sp.", filter)))
-        if "doc" in _root_keys(filter):
-            index = self._index(include_job_document=True)
-        else:
-            index = self._sp_index()
+        include_job_document = "doc" in _root_keys(filter)
+        index = list(self._build_index(include_job_document=include_job_document))
```

Comment:
Nit: could just inline the single-use flag:
```suggestion
        index = list(self._build_index("doc" in _root_keys(filter)))
```

Reply:
My rationale for the existing snippet was that giving a name `include_job_document` to the condition helped with self-documenting code. I also liked the use of a keyword argument in the `_build_index` call, and the use of a variable gave it slightly nicer formatting than the longer inlined statement which wraps (iirc) when used with a keyword argument.

Reply:
Resolved in 9457215f481648e39bf010cc3dd9a9001a197983.

Reply:
I agree with using the kwarg, so I'm in favor of the change you made. I prefer a linter's line-wrapping to an extraneous variable definition since the latter could change expectations for someone reading the code while formatting is clearly just formatting.